pkg/util/util.go

```diff
@@ -54,14 +55,14 @@ func TransformSize(n uint64) string {
 	case n < 1024:
 		return strconv.FormatUint(n, 10) + "B"
 	case n < 1048576 && n > 1024:
-		n = n / 1024
-		return strconv.FormatUint(n, 10) + "KiB"
+		tmp := float64(n) / 1024
```

Comment:
instead of this can't we have done the following:

```go
var tmp float64

tmp = n / 1024
```
that way we can avoid the unnecessary variable creation and typecasting, WYT?

Reply:
Even if we declare `tmp` as a `var`, we will need to cast because we are diving `int64` with an `int`.